router/pkg/config/config.schema.json

```diff
@@ -1706,6 +1706,17 @@
               "default": false
             }
           }
+        },
+        "truncate_floats":{
+          "type": "object",
+          "description": "Truncate floats like 1.0 to 1, 2.0 to 2, etc.",
```

Comment:
It should be clear that it only truncates `.0`. It does not truncate `1.1` for example.

Reply:
updated